website/docs/r/policy_lb_http_monitor_profile.html.markdown

```diff
@@ -0,0 +1,77 @@
+
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_lb_http_monitor_profile" "test" {
+  display_name          = "test"
+  description           = "Terraform provisioned LBHttpMonitorProfile"
+  request_body          = "test"
+  request_method        = "HEAD"
+  request_url           = "test"
+  request_version       = "HTTP_VERSION_1_1"
+  response_body         = "test"
+  response_status_codes = [200]
+  fall_count            = 2
+  interval              = 2
+  monitor_port          = 8080
+  rise_count            = 2
+  timeout               = 2
+
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `request_body` - (Optional) String to send as part of HTTP health check request body. Valid only for certain HTTP methods like POST.
+* `request_header` - (Optional) Array of HTTP request headers.
+  * `header_name` - (Optional) Name of HTTP request header
+  * `header_value` - (Optional) Value of HTTP request header
+* `request_method` - (Optional) Possible values are: `GET`, `OPTIONS`, `POST`, `HEAD`, `PUT`.
+* `request_url` - (Optional) For HTTP active healthchecks, the HTTP request url sent can be customized and can include query parameters.
+* `request_version` - (Optional) Possible values are: `HTTP_VERSION_1_0`, `HTTP_VERSION_1_1`, HTTP request version.
+* `response_body` - (Optional) If HTTP response body match string (regular expressions not supported) is specified (using LBHttpMonitor.response_body) then the healthcheck HTTP response body is matched against the specified string and server is considered healthy only if there is a match. If the response body string is not specified, HTTP healthcheck is considered successful if the HTTP response status code is 2xx, but it can be configured to accept other status codes as successful.
+* `response_status_codes` - (Optional) The HTTP response status code should be a valid HTTP status code.
+* `fall_count` - (Optional) Mark member status DOWN if the healtcheck fails consecutively for fall_count times.
+* `interval` - (Optional) Active healthchecks are initiated periodically, at a configurable interval (in seconds), to each member of the Group.
+* `monitor_port` - (Optional) Typically, monitors perform healthchecks to Group members using the member IP address and pool_port. However, in some cases, customers prefer to run healthchecks against a different port than the pool member port which handles actual application traffic. In such cases, the port to run healthchecks against can be specified in the monitor_port value. For ICMP monitor, monitor_port is not required.
```

Comment:
The sentence in the end about ICMP monitor should be removed from other types of monitors, as its not applicable to them